### ISSUE TITLE
Playback speed intervals and scaling time to playback speed

### ISSUE THREAD

**Issue by foinse** [prologueapp/Prologue#56]
One of the things I do like about the ABS phone app, is the playback time is scaled to the listening speed, so if you're listening to a 20 minute long chapter at 2x speed the app will show 10 minutes left in the chapter and similar for the length of the full book. Would it be possible to add an option for this?

Another thing, would it be possible to add smaller increments to the playback speeds? I usually listen at 1.9x but this isn't available as the increments are at .25 steps.

thanks!

**Comment by wesleyeff** [comment]
I agree about the small increments in playback speed. This is a regression from the old player where you could increment in 0.1 steps. 

Please allow smaller increments are allow us to customize the options in the speed picker.

**Comment by fauxpaux** [comment]
Agreed. I miss the previous fine controls on speed

**Comment by lcharlick** [comment]
The "[...] remaining in book" label on the now playing view is scaled with playback speed now. Also, playback speed is incremented in 0.05 steps. Closing this.

**Comment by foinse** [comment]
Thank you! Is there any chance the progress bar timings can be scaled too? if not thats ok. thanks for your work!
